web/router.js

```diff
@@ -242,6 +243,10 @@ router.beforeEach(async (to, from, next) => {
       }
     } else {
       next(false);
+
+      const event = analyticsClient.signInEvent();
```

Comment:
with several of these events, we've made deliberate judgments about what sequence they should occur in.  we should either document those judgments at these callsites, or in `AnalyticsClient` itself (as some normative advice on how to use this).